docs/configuration.md

```diff
@@ -302,11 +304,6 @@ config file, like so:
     line-length = 100
     ```
 
-All of the above rules apply equivalently to `pyproject.toml`, `ruff.toml`, and `.ruff.toml` files.
-If Ruff detects multiple configuration files in the same directory, the `.ruff.toml` file will take
-precedence over the `ruff.toml` file, and the `ruff.toml` file will take precedence over
-the `pyproject.toml` file.
-
```

Comment:
I prefered this phrasing because it is more specific about when the precedence applies (it applies on a per directory level)

Reply:
Would it work for you if we keep the exact wording just move the lines 306-308 up as is?